src/FirstAssignment/CharactersEqualOccurrences.java

```diff
@@ -1,7 +1,29 @@
 package FirstAssignment;
 
-
 public class CharactersEqualOccurrences {
 
+	public static void main(String[] args) {
+		boolean b1;
+		b1 = areOccurrencesEqual("abacbc");
```

Comment:
no need of b1 and if you need to use it, provide the meaningful name

Reply:
Removed b1 from code